packages/node/src/utils/hot-reload.js

```diff
@@ -13,7 +13,7 @@ export const revalidate = (options) => {
         }
         const name = property;
         const url = remote;
-        (global.webpackChunkLoad || fetch)(url)
+        (global.webpackChunkLoad || global.fetch || require('node-fetch'))(url)
```

Comment:
what if you prefer other lib?
for the meantime i think you can set as global.fetch=require('whatever')

Reply:
Sure, I could do that, but I added this because:
- the `module-federation/node` package already depends on `node-fetch` (see [here](https://github.com/module-federation/nextjs-mf/blob/4a5e2824400cc843fa0c0504936a68c6c9f33946/packages/node/package.json#L24))
- this fallback is already used in NodeFederationPlugin as I mentioned in the PR description

Reply:
solid argument!